tools/tcg_rim_tool/src/main/java/hirs/swid/utils/Commander.java

```diff
@@ -106,10 +106,15 @@ public final void parseArguments(final String[] args) {
                 	parse = true;
                 	parseFile = args[++i];
                 	break;
-                case FULL_COMMAND_PREFIX + KEY_STRING:
+/*
```

Comment:
Guess you can get rid of this commented out code?

Reply:
Comments deleted.